app/models/budget.rb

```diff
@@ -3,5 +3,9 @@ class Budget < ActiveRecord::Base
   has_many :incomes
   has_many :expenses
 
-  validates :name, presence: true, uniqueness: true
+  enum frequency: { weekly: 0, fortnightly: 1, monthly: 2 }
+
+  validates :name, presence: true
+  validates :frequency, presence: true
+  validates :frequency, inclusion: { in: Budget.frequencies.keys }
```

Comment:
Lines 9 and 10 can be merged to one line

Reply:
Thanks Elle, done in another PR, ad-budget-frequency-amounts, sha: 5ed2ca8